dot/parachain/network-bridge/sender.go

```diff
@@ -104,3 +107,48 @@ func (nbs *NetworkBridgeSender) processMessage(msg any) error {
 
 	return nil
 }
+
+const requestTimeout = 200 * time.Millisecond // TODO is this reasonable?
+
+// PoV is probably the largest message and is currently set at 5MB, but will likely be increased to 10MB in the future.
+// see: https://github.com/paritytech/polkadot-sdk/issues/5334
+// Maybe message types should have a MaxSize() method instead of using the same value for all messages.
```

Comment:
yeah, I would say you can attach to the protocol string the max response size

Reply:
The question then becomes what the values for requests other than `PoVFetchingV1` should be. Would you be ok with deferring this change to a later PR, given that using smaller values here amounts to a potential performance optimization?